### PR TITLE
[IMP] mail: remove ComposerTextInputComponent/_getSelectionEnd 

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2579306

### CODE REVIEW

File: addons/mail/static/src/components/composer_text_input/composer_text_input.js [hunk 1]
```diff
@@ -63,23 +63,13 @@ export class ComposerTextInput extends Component {
...
-     * Returns textarea current content.
-     *
-     * @private
-     * @returns {string}
-     */
+      * Returns textarea current content.
+      *
+      * @private
+      * @returns {string}
+      */
```
Comment (oomsveta) [inline addons/mail/static/src/components/composer_text_input/composer_text_input.js:70]: bad spacing

**Review by oomsveta — CHANGES_REQUESTED** [review]
runbot red 👺

**Comment by oomsveta** [comment]
conflicting files

**Comment by alexkuhn** [comment]
Changes done

**Review by oomsveta — APPROVED** [review]

**Pull request closed without merging**